app/analysis/edit/template.hbs

```diff
@@ -16,107 +16,173 @@
         </h1>
         <div class="card">
           <div class="card-block">
-            {{#with intersection.lastSuccessful.value.model as |model|}}
-              {{#validated-form
-                model     = (changeset model IntersectionValidations)
-                on-submit = (perform save)
-              as |f|}}
-                {{#task-selection
-                  on-set-customer = (action (queue (action (mut f.model.customer)) (action 'validate' f.model)))
-                  on-set-project  = (action (queue (action (mut f.model.project)) (action 'validate' f.model)))
-                  on-set-task     = (action (queue (action (mut f.model.task)) (action 'validate' f.model)))
-                  initial         = (hash
-                    customer      = _customer
-                    project       = _project
-                    task          = _task
-                  )
-                as |t|}}
-                  {{#f.input name='task' as |fi|}}
+            {{#let intersection.lastSuccessful.value.model as |model|}}
+              <ValidatedForm
+                @model={{changeset model IntersectionValidations}}
+                @on-submit={{perform save}}
+                as |f|
+              >
+                <TaskSelection
+                  @on-set-customer={{action (queue (action (mut f.model.customer)) (action 'validate' f.model))}}
```

Comment:
Im not sure but is the first `action` helper here needed or could you just leave it out since out wrap all the calls in `action` anyway?

Reply:
Good question.. but apparently it's needed: https://github.com/DockYard/ember-composable-helpers#queue

Reply:
Okay then.